script.js

```diff
@@ -1,4 +1,339 @@
+  }
+}
+
+// end of main
+
+// dealing initial hands small function.
+function dealInitialHands() {
+  for (let i = 0; i < 2; i++) {
+    playerHand.push(deck.pop());
+    computerHand.push(deck.pop());
+  }
+}
+
+//checking initial hand for blackjacks
+function checkInitialHands(playerHand, computerHand) {
+  let playerStartScore = 0;
+  let computerStartScore = 0;
+  let initialPhaseResult = "";
+  for (let i = 0; i < playerHand.length; i += 1) {
+    playerStartScore += playerHand[i].score;
+  }
+  for (let j = 0; j < computerHand.length; j += 1) {
+    computerStartScore += computerHand[j].score;
+  }
+  if (playerStartScore === computerStartScore && playerStartScore === 21) {
+    initialPhaseResult = "double blackjack";
+  } else if (playerStartScore === 21 && computerStartScore !== 21) {
+    initialPhaseResult = "player blackjack";
+  } else if (playerStartScore !== 21 && computerStartScore === 21) {
+    initialPhaseResult = "computer blackjack";
+  } else {
+    initialPhaseResult = "game continue";
+  }
+  return initialPhaseResult;
+}
+// function to reset the whole game
+function refreshEverything() {
+  gameState = "initialize";
+  deck = [];
+  deck = shuffleCards(makeDeck());
+  playerHand = [];
+  computerHand = [];
+  playerScore = 0;
+  computerScore = 0;
+  playerBustStatus = false;
+  computerBustStatus = false;
+}
+
+//function to display all players cards depending on current size of hand
+function callCards() {
+  let message = "";
+  for (i = 0; i < playerHand.length; i += 1) {
+    message += `<br>${playerHand[i].name} of ${playerHand[i].suit} [${playerHand[i].score}]`;
+  }
+  return message;
+}
+
+// function to display all cpu cards depending on current size of hand
+function callCPUCards() {
+  let message = "";
+  for (i = 0; i < computerHand.length; i += 1) {
+    message += `<br>${computerHand[i].name} of ${computerHand[i].suit} [${computerHand[i].score}]`;
+  }
+  return message;
+}
+
+//function to calculate player scores
+function calculatePlayerScore(playerHand) {
+  let score = 0;
+  for (let i = 0; i < playerHand.length; i += 1) {
+    score += playerHand[i].score;
+  }
+  return score;
+}
+
+//function to calculate computer scores
+function calculateComputerScore(computerHand) {
+  let score = 0;
+  for (let i = 0; i < computerHand.length; i += 1) {
+    score += computerHand[i].score;
+  }
+  return score;
+}
+//function to check for aces, and reduce ONLY one ace each time if there are multiple aces.
+function checkForAces(playerHand) {
+  let temporaryPlayerScore = calculatePlayerScore(playerHand);
+  if (temporaryPlayerScore > 21) {
+    for (let i = 0; i < playerHand.length; i += 1) {
+      if (playerHand[i].name === "Ace") {
+        playerHand[i].name = "🂡 Ace";
+        playerHand[i].score = 1;
+        break;
+      }
+    }
+  }
+}
+//output message for player turn
+function outputMessage() {
```

Comment:
Same here. I think you can combine this function together with the one below and take in an argument.